flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/utils/TableTestBase.scala

```diff
@@ -213,9 +224,14 @@ object TableTestUtil {
   }
 
   def streamTableNode(table: Table): String = {
-    val dataStreamTable = table.getQueryOperation.asInstanceOf[DataStreamQueryOperation[_]]
-    s"DataStreamScan(id=[${dataStreamTable.getDataStream.getId}], " +
-      s"fields=[${dataStreamTable.getTableSchema.getFieldNames.mkString(", ")}])"
+    val (id, fieldNames) = table.getQueryOperation match {
+      case q: SDataStreamQueryOperation[_] => (q.getDataStream.getId, q.getTableSchema
```

Comment:
Shall we rename `DataStreamQueryOperation` to `ScalaDataStreamQueryOperation` and `JavaDataStreamQueryOperation` to clearly make the difference?